airflow/www/decorators.py

```diff
@@ -104,6 +104,8 @@ def wrapper(*args, **kwargs):
 
                 params = {k: v for k, v in itertools.chain(request.values.items(), request.view_args.items())}
 
+                if params and "is_paused" in params:
+                    extra_fields.append(("is_paused", params["is_paused"] == "false"))
```

Comment:
Can you add a test for this fix

Reply:
Thanks for the feedback. Will do.

Reply:
please ping us once you do and we will take a look